Automattic/jetpack
# Add JetpackIframeEmbed lib

#### Changes proposed in this Pull Request:

* add new class to lib - `Jetpack_Iframe_Embed` that allows us to enhance previews for Calypso (more context p8F0Ux-m4-p2)

#### Testing instructions:

* apply this patch to your jetpack site
* go to https://wordpess.com/view/ and pick your site
* try clicking a few links, see if your site works correctly
* see if the preview toolbar in Calypso gets updated URL as you navigate
* there should be no adminbar/masterbar inside the preview


#### Proposed changelog entry for your changes:
Enhanced preview functionality for usage with Calypso.
